source/_integrations/holiday.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Holiday
+description: Instructions on how to use holiday calendars in Home Assistant.
+ha_category:
+  - Calendar
+ha_iot_class: Local Polling
+ha_release: 2023.12
+ha_config_flow: true
+ha_domain: holiday
+ha_platforms:
+  - calendar
+ha_codeowners:
+  - '@jrieger'
+ha_integration_type: integration
+---
+
+The Holiday integration allows you to create a calendar of holidays in Home Assistant for powering automations.
```

Comment:
Probably we should ref the upstream library like we do in [holiday](https://www.home-assistant.io/integrations/workday/)